### PR TITLE
Add gpt-4o metadata

### PR DESCRIPTION
**Pull request by ConnorDoyle**
#3612

## Type

🆕 New Feature
🐛 Bug Fix
🧹 Refactoring
📖 Documentation
💻 Development Environment
🚄 Infrastructure
✅ Test

## Pre-Submission Checklist (optional but appreciated):

- [ ] I have included relevant documentation updates (stored in /docs/my-website)

## OS Tests (optional but appreciated):

- [ ] Tested on Windows
- [ ] Tested on MacOS
- [ ] Tested on Linux

### CODE REVIEW

File: model_prices_and_context_window.json [hunk 1]
```diff
@@ -9,6 +9,18 @@
         "mode": "chat",
         "supports_function_calling": true
     },
+    "gpt-4o": {
```
Comment (lowjiansheng) [inline model_prices_and_context_window.json:12]: Can we add an alias for `gpt-4o-2024-05-13` too?

**Comment by mmmaia** [comment]
Thanks for working on this! A couple observations:

  - `max_output_tokens` for GPT-4o is 2048 (at least it's what I see in the playgronud)
  - we should also add `gpt-4o-2024-05-13`

**Comment by HowieG** [comment]
How is mode handled with multimodal input?
"mode": ["chat", "audio_transcription", "image_generation"],

**Comment by ConnorDoyle** [comment]
> Thanks for working on this! A couple observations:
> 
> * `max_output_tokens` for GPT-4o is 2048 (at least it's what I see in the playgronud)
> * we should also add `gpt-4o-2024-05-13`

I couldn't find docs supporting either explicit API limit, and I was unable to convince the model to produce more than 2048 tokens via some manual API testing. Updated this to 2048 since the limit is _at least_ 2048 via the API (but possibly higher.)

**Comment by mmmaia** [comment]
> I couldn't find docs supporting either explicit API limit, and I was unable to convince the model to produce more than 2048 tokens via some manual API testing. Updated this to 2048 since the limit is _at least_ 2048 via the API (but possibly higher.)

Now my playground is showing 4096. Unfortunately, there's no mention about it in the docs.

**Comment by ghost** [comment]
LGTM - merging in, we'll take care of updating the max token limit based on testing

**Comment by Undertone0809** [comment]
> How is mode handled with multimodal input? "mode": ["chat", "audio_transcription", "image_generation"],

#3629 So the question is, is it because of this?

**Comment by ghost** [comment]
@Undertone0809 mode is primarily used for vertex ai for routing between their different sdk's. 

We also started using it for health checks, as a way to know whether it's a `/completion`, `/embedding` or `/chat/completion` call 

No impact on multimodal.

**Pull request merged**